tests/components/component-playground/search-without-results/events/handlers.js

```diff
@@ -11,7 +11,7 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
   });
 
   it('should only render selected component based on search value', function() {
-    expect(component._getFilteredFixtures())
+    expect(component._getFilteredComponents())
```

Comment:
Never call private methods in tests, they're implementation details (as you can see, you had to update the tests when you refactored your code). Test your component only through its public API.